resources/definitions/gmax15plus.def.json

```diff
@@ -10,13 +10,16 @@
         "category": "Other",
         "file_formats": "text/x-gcode",
         "platform": "gmax_1-5_xt-plus_s3d_full model_150707.stl",
-        "has_machine_quality": true,
+		"has_machine_quality": true,
         "has_variants": true,
-        "variants_name": "Hotend",
-        "preferred_variant_name": "0.5mm E3D (Default)",
-        "machine_extruder_trains": {
+		"variants_name": "Hotend",
+		"preferred_variant_name": "*0.5mm E3D (Default)*",
```

Comment:
Remove `*`, we don't use wildcards for this setting. Otherwise when adding a new gMax printer to Cura, the variant will be empty.